CMakeLists.txt

```diff
@@ -119,3 +119,8 @@ install(
     DIRECTORY include/
     DESTINATION include
 )
+
+install(
+    FILES package.xml
```

Comment:
Can you please add
set (pfs_PACKAGE_ROS_XML package.xml)

Next to the other PACKAGE `set()` directives?